### DISCUSSION TITLE
scroll_area_in_tabs in release-7.0 branch

### DISCUSSION THREAD

**Discussion by ToppDev** [Adanos020/egui_dock#169] [Q&A] [answered]
Hello,

I wanted to switch to the `release-7.0` branch, because I like the work you did there with the windowing.
However I struggled to find `DockArea::scroll_area_in_tabs()`, from the 0.6 version.

Is there still a way to disable the ScrollArea? I need this because I embed a container where I handle scrolling myself, but the Scrollbar appears then which looks not so nice
![image](https://github.com/Adanos020/egui_dock/assets/17766093/671e466c-dfaf-4015-913d-c436739cc3d8)

Thanks in advance for your help.

**Comment by anhosh** [comment, marked as answer]
Hey! In a recently merged PR (#160), `DockArea::scroll_area_in_tabs` was replaced with `TabViewer::scroll_bars` which lets you control both scroll bars on individual tabs. If you want to disable all scrolling on all tabs, simply override that method and make it always return `[false, false]`.

This change is included in the changelog file on the `release-0.7` branch. I recommend checking it out from time to time.

Hope this helps!

**Reply by ToppDev** [reply]
Exactly what I was searching for. Thanks :+1:
